tests/console/check_system_info.pm

```diff
@@ -35,6 +39,23 @@ sub run {
         assert_script_run("grep -w $milestone_version /etc/issue");
     }
     assert_script_run('cat /etc/os-release');
+
+    my $myaddons = "";
+    $myaddons    = get_var('SCC_ADDONS');
+    $myaddons    = $myaddons . ",base,serverapp" if (is_sle('15+') && check_var('SLE_PRODUCT', 'sles'));
```

Comment:
Have you consider the bless list for SLE12? If the module on base which corresponding module dropped on target system, then you won't find it on target.

Reply:
Yes. This has been done at get_addon_fullname function

Reply:
Updated the code and wait the verify run